pkg/controller/velerobackup/velerobackup_controller.go

```diff
@@ -129,6 +146,22 @@ func (r *ReconcileBackup) Reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{}, err
 	}
 
+	// Check to see how long since the last back was taken (we may need to rate limit)
+	delta := metav1.Now().Sub(cp.Spec.LastBackupTime.Time)
+	if delta < reconcileRateLimitDuration {
+		// calculate the next time this reconcile should attempt to run. Add 1 second to avoid a race condition right
```

Comment:
There is no need to avoid this race condition. Let's not add extra code to avoid it.